README.md

```diff
@@ -39,8 +39,16 @@ And review the following guides:
 * [Configuring Remote Video Subscription](guides/configuring_remote_video_subscriptions.md)
 
 ## Include Amazon Chime SDK in Your Project
+You can download and integrate the Amazon Chime iOS SDKs to your project from both CocoaPods and binaries from Github release.
```

Comment:
You can integrate Amazon Chime SDK in your project from either CocoaPods or binaries through Github release.